.github/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## Release 0.4.1 (development release)
```

Comment:
Given that `0.3.1` is the latest release, we should rename this release to `0.3.2` and replace the empty `0.4.0` release notes below with these ones. Ideally, we would also set the version in `_version.py` to `0.3.2` to prepare the XCC for a new release.